### DISCUSSION TITLE
Is it possible to make single TextField scrollable?

### DISCUSSION THREAD

**Discussion by manupatel007** [flet-dev/flet#1989] [Q&A]
### Question

I have a TextField which can contain multiline text, even exceeding the window's height. Is it possible to add scroll behavior to it once it overflow the window? I have tried to contain it in Row and Column controls with scroll enabled, but no luck.

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment]
Can you paste a sample code please?

**Reply by manupatel007** [reply]
Yes Sure

`
import flet as ft

def main(page: ft.Page):
    page.title = "Flet Text Editor"

    txt_number = ft.TextField(text_align=ft.TextAlign.LEFT, expand=True, multiline=True, border="none", hint_text="Write your text here...")

    # lv = ft.ListView()
    page.add(
        ft.Column(
            [
                txt_number
            ],
            scroll=ft.ScrollMode.ALWAYS
        )
    )

ft.app(target=main)
`

**Reply by ndonkoHenri** [reply]
Solution: add an `expand=True` to the column too ( i will also advice to comment the `scroll` of the column, but you could leave it…)
```python
ft.Column(
    [
        txt_number
    ],
    expand=True,
    # scroll=ft.ScrollMode.ALWAYS
)
```

You can also completely leave out the `Column` (it's `scroll` prop is not necessary).
